src/reportWebVitals.js

```diff
@@ -1,15 +0,0 @@
-const reportWebVitals = (onPerfEntry) => {
```

Comment:
- [ ] You have done well removing this generated file. Kindly remove the following files as well in the `public` folder.

  ![mdkdie](https://github.com/devendra-alt/math-magicians/assets/81468675/c5985b0d-a2c3-4981-9e57-ac7517838462)

Reply:
thank you @julie-ify, for your valuable feedback 😄